tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -274,6 +275,10 @@ public void ProfilerLogOptions_ShouldCreateMlpdFiles ([ValueSource (nameof (Prof
 		[Test]
 		public void CustomLinkDescriptionPreserve ([Values (AndroidLinkMode.SdkOnly, AndroidLinkMode.Full)] AndroidLinkMode linkMode)
 		{
+			if (Builder.UseDotNet && linkMode == AndroidLinkMode.Full) {
+				// TODO: remove when merged https://github.com/xamarin/xamarin-android/issues/5288
```

Comment:
Opened https://github.com/xamarin/xamarin-android/pull/5294

That should hopefully fix the issue on .NET5 and we can start testing `Full` link mode again.